lib/android/app/src/main/java/com/reactnativenavigation/react/coordinator/CoordinatorLayoutView.java

```diff
@@ -0,0 +1,54 @@
+package com.reactnativenavigation.react.coordinator;
+
+
+import android.content.Context;
+import android.support.design.widget.AppBarLayout;
+import android.support.design.widget.CoordinatorLayout;
+import android.support.v4.widget.NestedScrollView;
+import android.util.AttributeSet;
+import android.util.Log;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.FrameLayout;
+
+public class CoordinatorLayoutView extends CoordinatorLayout {
+
+	public CoordinatorLayoutView(Context context) {
+		super(context);
+		init();
+	}
+
+	public CoordinatorLayoutView(Context context, AttributeSet attrs) {
```

Comment:
remove additional unused constructors, and inline the `init` method to be inside constructor